libcodechecker/libclient/thrift_helper.py

```diff
@@ -68,7 +68,7 @@ def wrapper(self, *args, **kwargs):
                     print('API call error: ' + funcName)
                     print(str(reqfailure))
 
-                raise
+                exit(0)
```

Comment:
Why is this change neccessary? Is it really a good idea to not let the client code who issued the Thrift req do any sort of error management or prompt the user to give a proper request (e.g. if run name is bogus or something)?